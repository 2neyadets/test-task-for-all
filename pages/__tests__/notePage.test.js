import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils'
import NotePage from '../note/index'

const localVue = createLocalVue()
localVue.use(Vuex)

describe('NotePage component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(NotePage, {
      localVue,
    })
  })

  it('mounts without errors', done => {
    localVue.config.errorHandler = done // для отлавливания ошибок внутри nextTick, если нужно что-то выпонять внутри него
    expect.assertions(1)
    expect(wrapper).toBeTruthy()
    done()
  })

  it('match snapshot', () => {
    expect.assertions(1)
    expect(wrapper.element).toMatchSnapshot()
  })
})
