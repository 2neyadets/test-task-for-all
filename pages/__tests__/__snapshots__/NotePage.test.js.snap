// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotePage component match snapshot 1`] = `
<errorlayout-stub
  error="[object Object]"
/>
`;
